server/utils/stripe.js

```diff
@@ -0,0 +1,70 @@
+const stripe = require('stripe')(process.env.STRIPE_SECERT);
+const Booking = require('../models/Booking');
+const Sitter = require('../models/Sitter');
+
```

Comment:
Good job making utils for this! Will definitely be beneficial as we roll out Stripe